### ISSUE TITLE
Menu bar "Edit" buttons don't work

### ISSUE THREAD

**Issue by jflasch** [AlexTorresSk/custom-electron-titlebar#84]
**Describe the bug**
The Edit buttons in the default menu: undo, redo, copy, paste, etc... do not work when using the custom title bar.

**To Reproduce**
1. Create a simple electron app with the sample code from this module to create a basic custom title bar.
2. Highlight some text in app, click on edit -> copy.
3. Try to paste this text somewhere, you will see that it was not copied to the clipboard.

You can do similar tests to the above with everything in the edit menu to see that it does not work.

**Expected behavior**
Edit submenu buttons should correctly interact with the electron app.

**Desktop:**
 - OS: Windows 10
 - Electron version 8.2.3
 - Node Version 10.12.0

**Comment by kanji-keraliya** [comment]
Same here. Any solution found?

**Comment by ekvedaras** [comment]
Can confirm. Not only the buttons don't work, but it hijacks they key bindings themselves, so there is no way to do copy/paste, select all, ctrl+r for reload, etc.

**Comment by ekvedaras** [comment]
Update: if you use the default menu items which include all the edit actions, it works.

**Comment by ekvedaras** [comment]
So as a simplest fix just use default menu and append your items:
```javascript
// Instead of
const menu = new remote.Menu()
// use this
const menu = remote.Menu.getApplicationMenu()
```

**Comment by minhloi** [comment]
This is because the **MenuItem** in the **renderer** process are different from those in the **main** process. I fixed this in pull request #121

**Comment by AlexTorresSk** [comment]
Thanks for you pull request 😊
